corehq/apps/app_manager/static/app_manager/js/detail-screen-config.js

```diff
@@ -1208,6 +1214,7 @@ hqDefine('app_manager/js/detail-screen-config.js', function () {
                                 spec.sortRows[j].field,
                                 spec.sortRows[j].type,
                                 spec.sortRows[j].direction,
+                                spec.sortRows[j].header[this.lang],
```

Comment:
If you override the `locale_id` method in the [Invisible](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/app_manager/detail_screen.py#L395-L395) class then you should be guaranteed that `self.column.format == 'invisible'` is always `True`